src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -196,14 +195,8 @@ function MoneyRequestConfirmPage(props) {
                         iouAmount={props.iou.amount}
                         iouComment={props.iou.comment}
                         iouCurrencyCode={props.iou.currency}
-                        onConfirm={(selectedParticipants) => {
-                            createTransaction(selectedParticipants);
-                            ReportScrollManager.scrollToBottom();
-                        }}
-                        onSendMoney={(paymentMethodType) => {
-                            sendMoney(paymentMethodType);
-                            ReportScrollManager.scrollToBottom();
-                        }}
+                        onConfirm={createTransaction}
+                        onSendMoney={sendMoney}
```

Comment:
ReportScrollManager.scrollToBottom is removed leaving

```
onConfirm={(selectedParticipants) => createTransaction(selectedParticipants)}
```

Reply:
Ah, that's because we can't have ReportScreenContext inside MoneyRequestConfirmPage. That's why I use `Report.notifyNewAction` approach to replace it.

Reply:
> Why it was added in the first place

So, previously all money request page (amount, participants, confirm) is placed inside MoneyRequestModal. What the PR https://github.com/Expensify/App/pull/17964 did is separate each page into its own screen. So, all the logic there is already been there before the PR.

> The changes included in this PR will break the rule in this comment, we will not have one source of truth

No, it won't. Whether we receive the event from Pusher, add a new comment from Report.addActions, or create a new money request, all of them use `notifyNewAction` that will trigger the callback which is our source of truth.

Reply:
Huh, so the PR is just a few months ago. I expected it about a year ago 😂.

> I am not sure why we didn’t go with this approach before

I guess it's much simpler using `ReportScrollManager.scrollToBottom`, but this is an interesting find.